internal/cmd/egctl/config.go

```diff
@@ -75,6 +79,12 @@ func allConfigCmd() *cobra.Command {
 
   # Retrieve full configuration dump with short syntax
   egctl c proxy all <pod-name> -n <pod-namespace>
+
+  # Retrieve all proxies configurations in a certain namespace
+  egctl config envoy-proxy all --proxies -n <pod-namespace>
```

Comment:
`proxies` seems redundant since the command already has `envoy-proxy` in it.
How about `-A/--all` which collects from all namespaces ? this doesnt solve the use case of all proxies in a namespace though :(  , lets find a solution to address both the use cases

Reply:
@Xunzhuo Any time to look at this recently and share your thoughts? :slightly_smiling_face:

Reply:
Maybe

```shell
egctl config envoy-proxy all -n $namespace
```

to get config from all the proxies in the namespace, and

```shell
egctl config envoy-proxy all --all-namespaces
```

to get from all proxies in all namespaces? That parallels `kubectl`...

Reply:
@zirain Hi, do you think we should continue to process this PR?

Reply:
+1 on the suggestion from @kflynn 

* get from a specific pod
```
egctl config envoy-proxy all -n <namespace> <pod>
```
* get from all pods in a namespace
```
egctl config envoy-proxy all -n <namespace>
```
* get from all pods in all namespaces
```
egctl config envoy-proxy all --all-namespaces/-A
```

Reply:
Thanks,done